.travis.yml

```diff
@@ -1,5 +1,35 @@
 language: node_js
 node_js:
-  - '7'
-  - '6'
-  - '4'
+  - 9
+  - 8
+  - 6
+  - 4
```

Comment:
These should be quoted. Semver versions are not numbers, even though major versions seems like it. Better to be explicit about it.

Reply:
Maybe. But I'm pretty sure they're just passed to `nvm` as is, so wether they're integers or strings won't make a difference.

Reply:
It will make a difference if someday we need `9.0.1` and forget to quote it. Just IMHO better to use the correct type for readability reasons. This is why I hate YAML.